filter.test.js

```diff
@@ -131,6 +141,20 @@ describe('Filter', () => {
 
       expect(result).toEqual(true)
     })
+//
+    it('should return true when at least one prefix matches the event', () => {
+      const filters = [
+        {ids: ['1'], kinds: [1], authors: ['a']},
+        {ids: ['4'], kinds: [2], authors: ['d']},
+        {ids: ['9'], kinds: [3], authors: ['g']}
+      ]
+
+      const event = {id: '987', kind: 3, pubkey: 'ghi'}
+
+      const result = matchFilters(filters, event)
+
+      expect(result).toEqual(true)
+    })
```

Comment:
let me know if this test makes sense or if you'd like other tests.

Reply:
I think most relays expect pairs of hex characters and will break if you pass an odd number.

Reply:
But I'm fine with this for the tests, it does the job, we can fix it later.